contrib/website/assets/styles/fonts.css

```
@@ -0,0 +1,360 @@
+/* cyrillic-ext */
+@font-face {
+    font-family: "Inter";
+    font-style: normal;
+    font-weight: 300;
+    font-display: swap;
+    src: url(https://fonts.gstatic.com/s/inter/v20/UcC73FwrK3iLTeHuS_nVMrMxCp50SjIa2JL7W0Q5n-wU.woff2) format("woff2");
+    unicode-range: U+0460-052F, U+1C80-1C8A, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
+}
+/* cyrillic */
+@font-face {
+    font-family: "Inter";
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)